store/tikv/client_batch.go

```diff
@@ -538,6 +549,7 @@ func (c *batchCommandsClient) initBatchClient() error {
 		return errors.Trace(err)
 	}
 	c.client = streamClient
+	c.client.Context().Done()
```

Comment:
What's the purpose of this change?

https://pkg.go.dev/google.golang.org/grpc?tab=doc#ClientStream

```
	// It should not be called until after Header or RecvMsg has returned. Once
	// called, subsequent client-side retries are disabled.
	Context() context.Context
```

Reply:
orz， my fault